src/bm_sim/actions.cpp

```diff
@@ -289,12 +291,18 @@ ActionFnEntry::execute(Packet *pkt) const {
 
   auto &primitives = action_fn->primitives;
   size_t param_offset = 0;
+  size_t si_offset = 0;
   // primitives is a vector of pointers
   BMLOG_TRACE_SI_PKT(*pkt, action_fn->get_source_info(),
-                     "Executing action {}", action_fn->get_name());
+                     "Action {}", action_fn->get_name());
   for (auto primitive : primitives) {
+    BMLOG_TRACE_SI_PKT(*pkt, action_fn->source_infos[si_offset],
```

Comment:
They can be disabled with '-L debug' instead of the current default '-L trace'.

Would you be interested in changing the default to '-L debug' so only people who explicitly set '-L debug' will see these new messages?

Or creating a new level higher than '-L trace' (although that might require changes to spdlog, which might not be desired).